### PR TITLE
Inpaint upscale fix

### PR DESCRIPTION
**Pull request by GuiAworld**
This messy code fixes the upscaling while using a mask. I'm not a developer, just really good at stitching different parts of code. 
It's best to examine my code and clean it up a bit.

Apparently, the upscale process did not properly updated img_orig (x0) or the mask.
So, I rescaled the original image to the new resolution, while keeping the original intact for multiple upscale passes.
And I changed the way the mask was rounded to keep the tensor size the same as the other tensors.

### CODE REVIEW

File: latent.ipynb [hunk 1]
```diff
@@ -988,8 +989,10 @@
         "            mask = Image.open(fetch(init_mask)).convert('RGB')\n",
         "            mask = TF.to_tensor(mask).to(device).unsqueeze(0)\n",
         "            if mask_rotate: mask = torch.rot90(mask, 1, [3,2])\n",
-        "            mask = F.interpolate(mask,[opt.H//8,opt.W//8]).mean(1)\n",
+        "            #mask = F.interpolate(mask,[opt.H//8,opt.W//8]).mean(1)\n",
+        "            mask = resize(mask,out_shape = [opt.n_samples,4,opt.H//8, opt.W//8])\n",
         "            mask = transform(mask)\n",
+        "            mask = torch.round(mask)\n",
```
Comment (Dango233) [inline latent.ipynb:995]: i intentionally left the mask as float. In this implementation, the float value controls the propotion of steps a pixel to be masked.
like, if a pixel value is 0.5, only the first 50% steps is masked. After that it will enter normal diffusion process.
